schema.yml

```diff
@@ -266,6 +374,33 @@ paths:
               schema:
                 $ref: '#/components/schemas/Database'
           description: ''
+  /api/db/v0/links/:
```

Comment:
@silentninja could you please review the spec for `links`, here?

Reply:
@silentninja I'm suggesting we close this PR. The `/links/` endpoint has already been merged into the develop branch and its API documentation is generated through another PR that is merged.